CONTRIBUTING.md

```diff
@@ -0,0 +1,179 @@
+   - Clone the repo you want to write to
+   - Create a new local branch from the main branch, name it after your contribution
+2. Uploading your Changes (iterative)
+   - Make your improvements
+   - Please try to write one sentence or clause per line, as far as it makes sense. This improves git diff and file reviews.
+   - Try to format your edited files with [Prettier] and lint Markdown with [markdownlint].
+   - commit to the new branch.
+   - Push the new branch to your remote
+3. Requesting the improvements to be merged
+   - On Github, create a pull request to merge the new branch into the main branch
+   - If the title is not sufficient, you can provide a PR description.
+   - Please leave the “allow edits” checkbox enabled
+4. Checking, cleaning (iterative)
+   - When a PR is created, Prettier runs on the PR branch.
+     If it detects unformatted files, a commit that applies formatting changes is added to the branch.
+     if you want to add more commits to the PR, you will have to pull these changes into your local branch.
+   - After prettifying is done, the Markdown files are checked for issues that relate to ambiguity, accessibility, consistency, and alike by [markdownlint].
+     If this check doesn't pass, you have to add commits to your branch which fix the detected issues.
+   - The checks rerun and the cycle repeats, until the checks succeed.
+5. Reviewing and improving content (iterative)
+   - Now your PR is ready for review, and you can request review from one or more collaborators.
+   - If a reviewer rejects your PR and provides feedback, you will have to discuss or integrate the feedback
+     by adding fixup commits to your PR branch until it gets approved.
+     This also triggers the checking and cleaning cycle.
+   - Meanwhile, if the main branch receives new commits and your PR can't be merged anymore due to merge conflicts,
+     please don't merge main into your branch, but rebase your branch on main and force-push your branch.
+     This keeps the commit history linear.
+     - Finally, a reviewer might approve your PR
+6. Merge
+   - The PR is clean, approved, and thus ready for merge.
+   - A maintainer will merge it.
+
+### Content file structure
+
+The content on this site is authored in Markdown, a lightweight markup language that is rendered to HTML.
+Page files are written in markdown or R markdown and organized in the content folder.
+The folder names represent the URL segments, and the menu is built from the folder structure.
+
+- Chapter pages are located at `content/<chapter-name>/_index.md`.
+  The underscore is important, and the metadata block should contain `chapter: true`.
+  Note that the text will be centered when `chapter:` is set to `true`.
+- Child pages are located either at `content/<chapter-name>/<page-name>/index.md` or at `content/<chapter-name>/<page-name>.md`.
+  If the former variant is used (page bundles), static assets can put directly into the page directory.
```

Comment:
please define or examplify static assets.

Reply:
Thanks, I've added some explanations and examples illustrating the usage of static assets.